release notes/v0.43.0.md

```diff
@@ -0,0 +1,125 @@
+
+This release brings a new experimental JavaScript module that adds [distributed tracing](https://en.wikipedia.org/wiki/Tracing_(software)) support to k6. It allows you to seamlessly instrument HTTP requests from k6, and, assuming the system you're testing is instrumented in the same way, would allow you unparalleled visibility of the SUT behavior for the lifetime of each request.
+
+Here's a simple example:
+
+```javascript
+import tracing from 'k6/experimental/tracing';
+import http from 'k6/http';
+
+tracing.instrumentHTTP({
+  propagator: 'w3c',
+});
+
+export default () => {
+  http.get('https://httpbin.test.k6.io/get', {
+    headers: {
+      'X-Example-Header': 'instrumented/get',
+    },
+  });
+};
+```
+
+You can see more details and examples in the [documentation](https://https://k6.io/docs/javascript-api/k6-experimental/tracing/).
+
+
+### `http.asyncRequest` [#2877](https://github.com/grafana/k6/pull/2877)
+
+The `k6/http` module has a new `asyncRequest` function that takes the same arguments as [`http.request()`](https://k6.io/docs/javascript-api/k6-http/request/), but returns a `Promise` that when `await`ed will be resolved with a `Response` object. This gives you more control over the script execution, as potentially the most time-consuming calls—making HTTP requests—will no longer block the thread of execution.
+
+See this example issuing a POST request:
+
+```javascript
+import http from 'k6/http';
+
+export default async function () {
+  const resPromise = http.asyncRequest(
+    'POST', 'https://httpbin.test.k6.io/post', { name: 'Bert' });
+  // Do something else here, make other requests, etc.
+  // Then when you're ready to use the response:
+  const resp = await resPromise;
+  console.log(resp.json().form.name); // Bert
+}
+```
+
+This is one of the first steps towards migrating our APIs to be asynchronous, and similar changes can be expected in the future.
+
+You can read more about `asyncRequest` in the [documentation](https://k6.io/docs/javascript-api/k6-http/asyncrequest/).
+
+
+### Enhancements and UX improvements
+
+- [#2754](https://github.com/grafana/k6/pull/2754), [#2805](https://github.com/grafana/k6/pull/2805) The output of the `k6 version` command has been enhanced to also show the version of all extensions built into the k6 binary produced by [xk6](https://github.com/grafana/xk6). Thanks, @HarrisChu!
+- [#2800](https://github.com/grafana/k6/pull/2800) Improved handling of the Ctrl+C signal to gracefully abort the test during VU initialization.
+- [#2803](https://github.com/grafana/k6/pull/2803) Ensure the REST API server is shut down after the test ends.
+- [#2867](https://github.com/grafana/k6/pull/2867) Added the ability to display test run details and logs from k6 Cloud.
+- [#2890](https://github.com/grafana/k6/pull/2890) Added a method for JavaScript modules to lookup environment variables without directly accessing the `os` package.
```

Comment:
```suggestion
- [#2890](https://github.com/grafana/k6/pull/2890) Added a method for JavaScript modules to lookup environment variables without directly accessing the `os` package.
- [#2910](https://github.com/grafana/k6/pull/2910) Added a bit more context when parsing the script options, so that it is more obvious what fails.
```